,0
0,I
1,am
2,new
3,to
4,both
5,python
6,and
7,django
8,.
9,I
10,am
11,trying
12,to
13,get
14,python
15,3
16,.4
17,and
18,django
19,1
20,.6.4
21,working
22,within
23,a
24,pyvenv
25,-3.4
26,environment
27,on
28,Linux
29,Mint
30,Mate
31,13
32,.
33,I
34,followed
35,the
36,procedures
37,of
38,Using
39,a
40,virtual
41,environment
42,with
43,Python
44,3
45,.4
46,for
47,"""Building"
48,Python
49,3
50,.4
51,from
52,source
53,""""
54,and
55,"""Using"
56,pyvenv
57,"-3.4""."
58,Then
59,following
60,the
61,Django
62,tutorial
63,at
64,Writing
65,your
66,first
67,Django
68,app
69,","
70,I
71,was
72,able
73,to
74,create
75,"""mysite"""
76,with
77,startproject
78,.
79,But
80,the
81,command
82,"""python"
83,manage
84,.py
85,runserver
86,""""
87,failed
88,with
89,django
90,.core.exceptions.
91,"ImproperlyConfigured: Error loading either pysqlite2 or sqlite3 modules
      (tried in that order)"
92,:
93,No
94,module
95,named
96,'_sqlite3'
97,Sorry
98,","
99,I
100,do
101,not
102,recall
103,why
104,I
105,resorted
106,to
107,compiling
108,from
109,source
110,","
111,but
112,it
113,may
114,be
115,that
116,I
117,was
118,unable
119,to
120,find
121,pyvenv
122,in
123,the
124,standard
125,Ubuntu
126,downloads
127,using
128,apt
129,-get.
130,In
131,any
132,case
133,","
134,the
135,Makefile
136,for
137,the
138,source
139,distribution
140,does
141,not
142,have
143,an
144,"""uninstall"""
145,target
146,","
147,so
148,it
149,seems
150,I
151,am
152,stuck
153,with
154,whatever
155,make
156,installed
157,","
158,and
159,without
160,a
161,plan
162,for
163,what
164,to
165,do
166,next
167,to
168,actually
169,get
170,pyvenv
171,-python-django
172,working
173,.
174,I
175,am
176,updating
177,this
178,post
179,by
180,attaching
181,the
182,full
183,error
184,:
